modules/log6x-6-1-0-rn.adoc

```diff
@@ -0,0 +1,39 @@
+= Logging 6.1.0 Release Notes
+// Module included in the following assemblies:
+//
+
+:mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-1-0_{context}"]
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:9038[{OCP-short}{logging} Release 6.1.0].
+
+[id="openshift-logging-release-notes-6-1-0-enhancements"]
+== New Features and Enhancements
+
+=== Log Collection
+
+* This enhancement adds the source `iostream` to the attributes sent from collected container logs. The value is set to either `stdout` or `stderr` based on how the collector received it. (link:https://issues.redhat.com/browse/LOG-5292[LOG-5292])
+
+* With this update, the default memory limit for the collector increases from 1024Mi to 2024Mi. Users should adjust resource limits based on their cluster’s specific needs and specifications. (link:https://issues.redhat.com/browse/LOG-6072[LOG-6072])
+
+* With this update, users can now set the syslog output delivery mode to either AtLeastOnce or AtMostOnce. (link:https://issues.redhat.com/browse/LOG-6355[LOG-6355])
+
+=== Log Storage
+
+* With this update, the new 1x.extra-small LokiStack size supports OCP clusters with fewer workloads and lower log volumes (up to 50GB/day). (link:https://issues.redhat.com/browse/LOG-5939[LOG-5939])
+
+[id="logging-release-notes-6-1-0-technology-preview-features"]
+== Technology Preview Features
+
+* This enhancement adds the originating `iostream` to the attributes forwarded from collected container logs. The value will be set to either `stdout` or `stderr` depending on how it was ingested by the collector. (link:https://issues.redhat.com/browse/LOG-5637[LOG-5637])
```

Comment:
This description is incorrect as the link as for e2e OTLP support an not the iostream feature which was properly listed  as an enhancement